frontend/www/js/omegaup/components/homepage/Carousel.vue

```diff
@@ -1,10 +1,14 @@
 <template>
   <!-- id-lint off -->
-  <div class="carousel slide" data-ride="carousel" id="carousel-display">
+  <div
+    class="carousel slide main-carousel"
+    data-ride="carousel"
+    id="carousel-display"
+  >
     <!-- id-lint on -->
     <ol class="carousel-indicators">
       <li
-        data-target=".carousel"
+        data-target=".main-carousel"
```

Comment:
esto funciona?

```suggestion
        data-target="#carousel-display"
```

si sí, creo que prefiero que se use el ID como selector.